packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -116,8 +121,9 @@ class App extends React.Component {
   }
 
   render() {
-    const Feature = this.state.feature;
-    return Feature ? <BuiltEmitter><Feature /></BuiltEmitter> : null;
+    const { feature } = this.state
+    if (feature !== null) return <BuiltEmitter feature={feature} />
```

Comment:
@gaearon Would you still like all features turned into classes or is this sufficient (see above, using `createElement` -- allows the `prototype` check)?